src/EnergyPlus/StandardRatings.cc

```diff
@@ -1955,31 +1967,38 @@ namespace StandardRatings {
                 if (BuildingCoolingLoad <= CoolingCapacityLS) {
                     PartLoadRatio = min(1.0, BuildingCoolingLoad / CoolingCapacityLS);
                     NetTotCoolCapBinned = PartLoadRatio * CoolingCapacityLS;
-                    PartLoadFraction = 1.0 - CyclicDegradationCoeff * (1.0 - PartLoadRatio);
+                    PartLoadFraction = CurveValue(PLFFPLRCurveIndex(spnum), PartLoadRatio);
+                    PartLoadFractionDefault = 1.0 - CyclicDegradationCoeff * (1.0 - PartLoadRatio);
                     TotCoolElecPowerBinned = (PartLoadRatio / PartLoadFraction) * CoolingElecPowerLS;
+                    TotCoolElecPowerBinnedDefault = (PartLoadRatio / PartLoadFractionDefault) * CoolingElecPowerLS;
```

Comment:
added local variables for SEER value calculation using the user specified PLF curve for multi-speed DX cooling coils.